Content.Server/Nyanotrasen/Mail/MailSystem.cs

```diff
@@ -178,9 +197,15 @@ private void OnAfterInteractUsing(EntityUid uid, MailComponent component, AfterI
 
             if (!HasComp<EmaggedComponent>(uid))
             {
-                if (idCard.FullName != component.Recipient || idCard.JobTitle != component.RecipientJob)
+                //if (idCard.FullName != component.Recipient || idCard.JobTitle != component.RecipientJob)
+                //{
+                //    _popupSystem.PopupEntity(Loc.GetString("mail-recipient-mismatch"), uid, args.User);
+                //    return;
+                //}
+
+                if (idCard.FullName != component.Recipient) // Frontier - Only match the name
```

Comment:
if we are stripping out the job differences, then the loot tables need to be drastically changed.

What you recieve in the mail is wholly dependant on your job at the time of the mail spawning.

Reply:
I see, so I might need to go around it and keep the rewards as per rank but still allow the open?
Ill redo it

Reply:
This is done on the mail spawn itself, this test just skip the open test when you tap pda/id on it

Reply:
oh true